distributed/bokeh/scheduler.py

```diff
@@ -19,7 +19,7 @@
 from bokeh.plotting import figure
 from bokeh.palettes import Viridis11
 from bokeh.io import curdoc
-from toolz import pipe
+from toolz import frequencies, pipe
```

Comment:
We should use cytoolz here for performance reasons.  

```python
try:
    from cytoolz import ...
except ImportError:
    from toolz import ...
```

Reply:
Or not, on the benchmark below it only helps by 20-30%, which still isn't good enough to continue with that diagnostic as is.